test/common/ops/scoreTask.test.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-shadow */
+import scoreTask from '../../../common/script/ops/scoreTask';
+import {
+  generateUser,
```

Comment:
these are unit tests, you should not import helpers from the integration tests. These will save the objects to the server which is not necessary.

You can use the helpers here https://github.com/piousbox/habitrpg/blob/shared-code-user-score-task/test/helpers/api-unit.helper.js#L16 which are not async and just create the necessary objects